main.py

```diff
@@ -5,34 +5,35 @@
 import discord  # installiere mit "python3 -m pip install -U discord.py" (MacOs/Linux) oder "py -3 -m pip install -U discord.py" (Windows)
 import random
 import asyncio
-import math
-from numpy import diff
 from discord import FFmpegPCMAudio  # nur für voice chat benötigt, muss vielleicht unabhängig installiert werden
 from os import listdir
 from os.path import isfile, join
 import os
-from matheAufgaben import createTask    # erstellt Aufgaben für das Kopfrechenspiel
+from matheAufgaben import createTask  # erstellt Aufgaben für das Kopfrechenspiel
 from commands import rnd_list, commandDescriptions
+from dotenv import load_dotenv
 
-intents = discord.Intents.default()     
-intents.message_content = True
-client = discord.Client(intents=intents)    # standardmäßige Berechtigungen, um Nachrichten zu senden
```

Comment:
die intents müssen enthalten bleiben, sonst ist der Bot nicht in der Lage, Nachrichten zu lesen und darauf zu reagieren